docs/en/setup/backend/aws-firehose-receiver.md

```diff
@@ -0,0 +1,21 @@
+# AWS Firehose receiver
+
+AWS Firehose receiver listen `127.0.0.1:13800` by default, and provides an HTTP Endpoint `/aws/firehose/metrics` that follows [Amazon Kinesis Data Firehose Delivery Stream HTTP Endpoint Delivery Specifications](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html)
+You could leverage the receiver to collect [AWS CloudWatch metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/working_with_metrics.html), and analysis it through [MAL](../../concepts-and-designs/mal.md) as the receiver bases on [OpenTelemetry receiver](./opentelemetry-receiver.md)
+
+## Setup(S3 example)
```

Comment:
We will have a more specific doc for s3, right?

Reply:
Yep, this doc introduces how the receiver works in common way,  S3 (or other AWS service) metrics doc should be provided in itself PR